### DISCUSSION TITLE
Separate buton for AP

### DISCUSSION THREAD

**Discussion by Carlos4x4** [calibx/msfsdeck#109] [Q&A] [answered]
Hello team,

I wanted to ask if you can separately implement the buttons to control the autopilot.

The "Folder" folder contains "AP" and "Airliner", which contain buttons to activate and deactivate the various Autopilot functions and show their status, but these windows are not customizable.

The idea is that these buttons are available separately in the "AP" folder so that we can design our own Autopilot windows.

Regards,
Carlos A.

**Comment by calibx** [comment, marked as answer]
Hi,

Great idea, I will try to add that in the tomorrow release.

**Reply by Carlos4x4** [reply]
Great thank you very much.

**Reply by calibx** [reply]
The 1.0.4 is out with separated inputs

**Reply by Carlos4x4** [reply]
Hi Calibx,
it would be great if in the next version you could incorporate the rest of the autopilot buttons like FD, APR, etc.
I mean the buttons on the "Airliner" window.
:-)

**Reply by calibx** [reply]
Ok I will add them too

**Comment by Carlos4x4** [comment]
Great news.
